hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CustomFeesValidator.java

```diff
@@ -182,8 +183,9 @@ private void validateExplicitTokenDenomination(
             @NonNull final WritableTokenStore tokenStore) {
         final var denomToken = tokenStore.get(tokenNum);
```

Comment:
I believe if you use `getIfUsable` here, we don't need line 188 🤔

Reply:
`getIfUsable` is not appropriate here because we need this to fail with `INVALID_TOKEN_ID_IN_CUSTOM_FEES` instead of `INVALID_TOKEN_ID` or `TOKEN_IS_PAUSED`

Reply:
Makes sense. There is another way we could do this, though: the account `getIfUsable` method has overrides for passing in the error response code. Maybe we add token overrides here as well, i.e.

TokenHandlerHelper.java:
```java
public Token getIfUsable(@NonNull final TokenID tokenId, @NonNull final ReadableTokenStore tokenStore, @NonNull final ResponseCodeEnum errorForDelete, @NonNull final ResponseCodeEnum errorForPause) {
        // other code
        // ....

        final var token = tokenStore.get(tokenId);
        validateTrue(token != null, INVALID_TOKEN_ID);
        validateFalse(token.deleted(), errorForDelete);
        validateFalse(token.paused(), errorForPause);
        return token;
}

public Token getIfUsable(@NonNull final TokenID tokenId, @NonNull final ReadableTokenStore tokenStore) {
        return getIfUsable(tokenId, tokenStore, TOKEN_WAS_DELETED, TOKEN_IS_PAUSED);
}

```